llvm/lib/Target/LoongArch/MCTargetDesc/LoongArchAsmBackend.cpp

```diff
@@ -226,11 +226,8 @@ bool LoongArchAsmBackend::shouldInsertFixupForCodeAlign(
       MCFixup::create(0, Dummy, MCFixupKind(LoongArch::fixup_loongarch_align));
```

Comment:
lld is not coupled with MC. Changes to both places are usually done in two patches.

Reply:
But `llvm-mc` is used in lld's tests. If `llvm-mc` is changed, lld's test will fail (i.e. `lld/test/ELF/loongarch-relax-emit-relocs.s`).